tools/cli/air/src/trace/run.rs

```diff
@@ -215,42 +229,38 @@ pub(crate) fn run(args: Args) -> anyhow::Result<()> {
     Ok(())
 }
 
-#[cfg(feature = "wasm")]
-fn get_runner(
+fn create_runner(
     mode: Option<Mode>,
-    air_interpreter_wasm_path: &Path,
+    _air_interpreter_wasm_path: &Path,
     _air_contract_wasm_path: &Path,
-    max_heap_size: Option<u64>,
+    _max_heap_size: Option<u64>,
 ) -> anyhow::Result<Box<dyn AirRunner>> {
-    let mode = mode.unwrap_or(Mode::Wasm);
+    let mode = mode.unwrap_or(default_mode());
     match mode {
         Mode::Native => {
             self::native::create_native_avm_runner().context("Failed to instantiate a native AVM")
         }
-        Mode::Wasm => self::wasm::create_wasm_avm_runner(air_interpreter_wasm_path, max_heap_size)
-            .context("Failed to instantiate WASM AVM"),
+        #[cfg(feature = "wasm")]
+        Mode::Wasm => {
+            self::wasm::create_wasm_avm_runner(_air_interpreter_wasm_path, _max_heap_size)
+                .context("Failed to instantiate WASM AVM")
+        }
         #[cfg(feature = "near")]
         Mode::Near => self::near::create_near_runner(_air_contract_wasm_path)
             .context("Failed to instantiate NEAR AVM"),
+        #[cfg(feature = "risc0")]
+        Mode::Risc0 => Ok(Box::new(self::risc0::Risc0Runner::new())),
     }
 }
 
 #[cfg(not(feature = "wasm"))]
-fn get_runner(
-    mode: Option<Mode>,
-    _air_interpreter_wasm_path: &Path,
-    _air_contract_wasm_path: &Path,
-    _max_heap_size: Option<u64>,
-) -> anyhow::Result<Box<dyn AirRunner>> {
-    let mode = mode.unwrap_or(Mode::Native);
-    match mode {
-        Mode::Native => {
-            self::native::create_native_avm_runner().context("Failed to instantiate a native AVM")
-        }
-        #[cfg(feature = "near")]
-        Mode::Near => self::near::create_near_runner(_air_contract_wasm_path)
-            .context("Failed to instantiate NEAR AVM"),
-    }
+fn default_mode() -> Mode {
```

Comment:
If it was a constant, it would be more straightforward.

Reply:
I don't have any special opinion here, let it be a constant